app/assets/javascripts/discourse/app/widgets/post-cooked.js

```diff
@@ -95,6 +95,26 @@ export default class PostCooked {
       return;
     }
 
+    // find the best a in each onebox and display link counts only for
+    // that one
+    const bestElements = [];
```

Comment:
I think this was supposed to be an object? 🙂 (or a `Map` even, though that would require Ching the code below)